css/custom.css.sample

```diff
@@ -1,11 +1,18 @@
-@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap');
+/* Magic Mirror Custom CSS Sample
+ *
+ * MIT Licensed.
+ */
 
+// Uncomment and adjust accordingly if you want to import another font from the google-fonts-api:
+// @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;700&display=swap');
+
+// Change color and fonts here:
```

Comment:
// This is js comment not css
/* This is css comment */

Reply:
Thanks for the info, will update the sample